includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -966,33 +968,49 @@ public static function print_validation_errors_meta_box( $post ) {
 		} );
 		</script>
 
+		<?php
+		$forced_sanitized_count = 0;
```

Comment:
Since the notice is displayed if `$forced_sanitized_count` is bigger than 0, we could display the notice as soon as a `$validation_error['forced_sanitization']` is found and `break` to avoid unnecessarily continuing the loop.

Reply:
Originally I showed a different message when _all_ of the sanitizations were forced vs _some_. So yes, this could `break` now.